geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -6392,6 +6401,23 @@ void basicDestroy(final EntryEventImpl event, final boolean cacheWrite, Object e
   }
 
   /**
+   * A set operation now can bootstrap a transaction now. User need to specifically disable this by
+   * setting this system property to true to get the old behavior.
+   *
+   * @Since Geode 1.3.0
+   */
+  public static final String PREVENT_SET_OP_BOOTSTRAP_TRANSACTION =
+      "preventSetOpBootstrapTransaction";
+
+  protected static boolean isSetOpBootstrapTransactionDisabled() {
+    return Boolean
```

Comment:
I think what you have ok to commit now and refactor it later, or you could do a little more work now that sets the precedent for how others will add future system properties.

If you decide to go the extra step now... then create a new class (maybe in org.apache.geode.internal.lang) called SystemProperty or something like that. Put the code in there to attempt to look up both "gemfire." and "geode." versions of the system property. Have the the code check and prefer "geode." first and only bother checking for "gemfire." if the "geode." system property is not set.